lib/private/Preview/Movie.php

```diff
@@ -9,11 +9,14 @@
 
 use OCP\Files\File;
 use OCP\Files\FileInfo;
+use OCP\IConfig;
 use OCP\IImage;
 use OCP\Server;
 use Psr\Log\LoggerInterface;
 
 class Movie extends ProviderV2 {
+	private Iconfig $config;
```

Comment:
```suggestion
	private IConfig $config;
```

Interfaces names are not case-sensitive but it’s still better to use the right one, it will help future devs when searching in the code. Static analysis may complain as well.

Reply:
Fixed. Learning a lot from this process, thank you everyone!